Update Ruby version to 2.7.2

SpaYco opened the pull request:
As requested by @nidalaa, I've updated the Ruby version in the Gemfile to 2.7.2, I've also updated the README.md so it puts the right information in the Ruby version.

SpaYco committed:
Update Ruby version to 2.7.2

SpaYco committed:
Update Ruby version to 2.7.2 in the README.md

bolah2009 reviewed (commented):
LGTM ... I think there's one more place to do the update. Also, have you tried testing if the app still works well with the version bump?

https://github.com/microverseinc/ror-social-scaffold/blob/6a3056610b0b03ba9357b3c0c672d518d08761f8/.ruby-version#L1

SpaYco commented:
will do!

SpaYco committed:
Update Ruby version to 2.7.2 in .ruby-version

SpaYco committed:
Update Gemfile.lock with latest packages

SpaYco commented:
@bolah2009 I updated `.ruby-version`, and tested it and it's working fine!

I also updated Gemfile.lock, since it causes some errors during installs.

euqueme commented:
Thank you @Spayco I also think this is good to go :ok_hand: @bolah2009 what do you say?
